docs/security.md

```diff
@@ -368,5 +368,12 @@ launch -Dapex.attr.SSL_CONFIG="{\"keyStorePath\":\"/opt/apexCli/ssl/myapex-keyst
 
 Apec CLI will copy the keystore file `/opt/apexCli/ssl/myapex-keystore.jks` to the destination STRAM node and also pass on the keystore password values to the STRAM. As mentioned above, you can also use configuration files to supply the value of `apex.attr.SSL_CONFIG`.
```

Comment:
It would be good to mention that this problem can also be solved without YARN patch by including ssl-server.xml into the classpath, such as via `apex.attr.LIBRARY_JARS`.